feature/experimental/qos/ate_tests/wrr_traffic_test/wrr_traffic_test.go

```diff
@@ -1714,7 +1509,7 @@ func ConfigureCiscoQos(t *testing.T, dut *ondatra.DUTDevice) {
 		targetGrpoup string
 	}{{
 		desc:         "scheduler-policy-BE1",
-		sequence:     uint32(6),
+		sequence:     uint32(1),
```

Comment:
Can we define human-friendly constants for these magic seq numbers: 0, 1, 4, 16?

Reply:
We can have , but general convention followed is for priority queues we use zero and one for non priority queues

Reply:
The OpenConfig MANDATES that sequence 0 is served 1st. And only when queue is empty (or PIR is achived) secheduler 1 is served. And so on.

So it is not convention. It's "standard".